### ISSUE TITLE
Choose Obstacles: Feedback

### ISSUE THREAD

**Issue by Gurkenator** [raithos/xwing#893]
### Expected Behavior

Obstacles (under the choose obstacles header) show the packs they're available from.

### Actual Behavior

Obstacles are labeled by only their 1.0 source. Adding a label showing the obstacle sources (never tell me the odds, Seps/Republic Squad pack etc.) like there is for cards would be ideal, since you wouldn't have to change the names people are used to in the selector. This could go hand in hand with adding another slot for environment cards, since you wouldn't have to look them up either then.

### Language

All

**Comment by raithos** [comment]
yeah let me think about this, right now theres no structure around this

**Comment by Patschke** [comment]
I don't see any principal issues to: 
  * include obstacle sources in the manifest
  * Display a list of sources below the image of the obstacle in the choice menu (there is empty space)
  * include checks for availability of the chosen obstacles in `isSquadPossibleWithCollection`, so we can display the squad-not-possible-with-collection warning where appropriate
 
 I'm not sure how and if we want to include environment cards - they aren't really part of a squad after all.

**Comment by raithos** [comment]
i also have to figure out the xws integration as well, afaik they still use the 1.0 naming for that as well.

**Comment by Patschke** [comment]
I don't see any reason why we would change the naming scheme, it's just some additional information that we could include? I was also thinking about including sources information for quickbuild cards as well every now and then.

**Comment by Gurkenator** [comment]
Sure. I've decided to include both 1.0 and 2.0 sets in the list. Apparently forces of mandalore will also include some debris clouds and asteroids. The 2.0 Decimator Expansion as far as I know doesnt include any obstacles, so if you decide to include 1.0 products in the list, make sure that the Decimator expansions dont get confused.

1.0 Core Asteroids
0 Old Core, NTMTO Obstacles Pack
1 Old Core, NTMTO Obstacles Pack
2 Old Core, Second Edition Core
3 Old Core, NTMTO Obstacles Pack
4 Old Core, Second Edition Core
5 Old Core, Second Edition Core
FA Asteroids
0 FA Core, NTMTO Obstacles Pack, 
1 FA Core, NTMTO Obstacles Pack, 
2 FA Core, NTMTO Obstacles Pack, 
3 FA Core, NTMTO Obstacles Pack, 
4 FA Core, NTMTO Obstacles Pack, 
5 FA Core, NTMTO Obstacles Pack, 
Debris Clouds
0 YT2400 (1.0), NTMTO Obstacles Pack
1 YT2400 (1.0), NTMTO Obstacles Pack
2 YT2400 (1.0), Second Edition Core
0 Decimator (1.0), NTMTO Obstacles Pack
1 Decimator (1.0), Second Edition Core
2 Decimator (1.0) Second Edition Core
Gas Clouds
0 Servants of strife squadron pack, guardians of the republic squadron pack
1 Servants of strife squadron pack, guardians of the republic squadron pack
2 Servants of strife squadron pack, guardians of the republic squadron pack
3 Never tell me the odds obstacles pack
4 Never tell me the odds obstacles pack
5 Never tell me the odds obstacles pack

**Comment by Patschke** [comment]
Thanks for the list! I've added them in, the obstacle selection now looks like that (in my development version of YASB - it's not live yet): 

![image](https://user-images.githubusercontent.com/17853362/145356697-34aa6e74-b940-4bdf-ba61-43fe6231ae64.png)

I've not added obstacles to the is-squad-possible-with-collection checks and did not add a tab to the collection stuff to allow for adding individual obstacles here. I've also not color-coded the obstacles choice menu to indicate if an obstacle is available. If there is feedback regarding this I might do so somewhen in the future, but not for now. 

I did not also add any ancient epic-unique obstacles without rules associated :D
